Algorithm.CSharp/UniverseSharingSubscriptionTradableRegressionAlgorithm.cs

```diff
@@ -111,29 +111,29 @@ public override void OnSecuritiesChanged(SecurityChanges changes)
             {"Total Trades", "1"},
             {"Average Win", "0%"},
             {"Average Loss", "0%"},
-            {"Compounding Annual Return", "75.079%"},
-            {"Drawdown", "2.200%"},
+            {"Compounding Annual Return", "69.904%"},
+            {"Drawdown", "2.000%"},
```

Comment:
Believe these stats changed because this algorithm also reproduces the underlying issue, and after the fix it enters the trade a day later.
Let's please add an explicity check in the 3 algorithms `OnData()` throwing if they get a duplicate time

Reply:
Agreed, will go ahead and add those checks!